lib/movie.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class Movie {
+  final String _title;
+  final String _originalTitle;
+  final String _overview;
+  final DateTime _releaseDate;
+  final double _voteAverage;
+  final List<String> _genres;
+  final Image _poster;
+  final Image _backdrop;
+  int _likes = 0;
```

Comment:
better to not have this underscores on the parameters of the constructor

Reply:
done